beacon_node/network/src/network_beacon_processor/sync_methods.rs

```diff
@@ -170,21 +172,20 @@ impl<T: BeaconChainTypes> NetworkBeaconProcessor<T> {
             if reprocess_tx.try_send(reprocess_msg).is_err() {
                 error!(self.log, "Failed to inform block import"; "source" => "rpc", "block_root" => %hash)
             };
-            if matches!(process_type, BlockProcessType::SingleBlock { .. }) {
```

Comment:
Why are we removing this match? 

It it redundant because we now only process single blocks here?

Reply:
Yes, blobs are processed in `process_rpc_blobs`